packages/solid-crs-presentation/lib/app.events.ts

```diff
@@ -56,11 +56,11 @@ export class ErrorEvent implements EventObject {
 /**
  * An event which is dispatched when this pod should be set up as an instition pod
  */
-export class SetProfileEvent implements EventObject {
+// export class SetProfileEvent implements EventObject {
 
-  public type: AppEvents.SET_PROFILE = AppEvents.SET_PROFILE;
```

Comment:
feel free to remove completely, was used in manage after login, but no login exists in presentation